### PR TITLE
[PSGP] Content Created

### PR DESCRIPTION
**Pull request by MeriadarGoblin**
Adding Planeshifted Guide to Phyrexia

### CODE REVIEW

File: planeshifted/planeshifted-guide/phyrexia/race-compleated.xml
Comment (FreshRoyal) [inline planeshifted/planeshifted-guide/phyrexia/race-compleated.xml:1]: Should probably include size selection

File: planeshifted/planeshifted-guide/phyrexia/race-elf.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			<p class="indent"><strong><em>Darkvision.</em></strong> You can see in dim light within 60 feet of you as if it were bright light and in darkness as if it were in dim light. You discern colors in that darkness only as shades of gray.</p>
+			<p class="indent"><strong><em>Keen Senses.</em></strong> You have proficiency in the Perception skill.</p>
+			<p class="indent"><strong><em>Iron Scent.</em></strong> As an action, you can pinpoint, by scent, the location of ferrous metal within 30 feet of you.</p>
+			<p class="indent"><strong><em>Mask of the Wild.</em></strong> You can attempt to hide even when you are only lightly obscured by foliage, heavy rain, falling snow, mist, and other natural phenomena.</p>
+			<p class="indent"><strong><em>Siege Monster.</em></strong> You deal double damage to objects and structures. When you damage a Construct creature with an attack or a spell, the attack or spell deals extra damage to the creature equal to your proficiency bonus.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Grants" id="ID_INTERNAL_RACIAL_TRAIT_TYPE_HUMANOID" />
```
Comment (FreshRoyal) [inline planeshifted/planeshifted-guide/phyrexia/race-elf.xml]: None of the humanoids implemented in the Legacy repo have this grant. I can see it was probably intended to have the elf grant

File: planeshifted/planeshifted-guide/phyrexia/race-compleated.xml
Comment (MeriadarGoblin) [reply planeshifted/planeshifted-guide/phyrexia/race-compleated.xml:1]: It is under Ancestral Legacy

File: planeshifted/planeshifted-guide/phyrexia/race-compleated.xml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+			<set name="category">Additional Feature</set>
+			<set name="slot">misc</set>
+			<set name="inventory-hidden">false</set>
+		</setters>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL_ALL_WILL_BE_ONE" />
+			<grant type="Racial Trait" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL_STRIKE" />
+			<grant type="Language" id="ID_PSLE_PSGP_LANGUAGE_PHYREXIA" />
+		</rules>
```
Comment (FreshRoyal) [inline planeshifted/planeshifted-guide/phyrexia/race-compleated.xml:152]: These should be granted by the race itself. Granting an item in a race doesn't work.
Remember to apply TCoE rules to the language as well.

File: planeshifted/planeshifted-guide/phyrexia/race-compleated.xml [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+	</element>
+	<element name="All Will Be One" type="Racial Trait" source="Planeshifted Guide to Phyrexia" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL_ALL_WILL_BE_ONE">
```
Comment (FreshRoyal) [inline planeshifted/planeshifted-guide/phyrexia/race-compleated.xml:154]: IDs also need to be changed

**Pull request merged**